sklearn/feature_selection/_univariate_selection.py

```diff
@@ -288,6 +290,189 @@ def chi2(X, y):
     return _chisquare(observed, expected)
 
 
+def _get_entropy(prob):
+    t = np.log2(prob)
+    t[~np.isfinite(t)] = 0
+    return np.multiply(-prob, t)
+
+
+def _info_gain(X, y, aggregate, ratio=False):
+
+    def _a_log_a_div_b(a, b):
+        with np.errstate(invalid="ignore", divide="ignore"):
+            t = np.log2(a / b)
+        t[~np.isfinite(t)] = 0
+        return np.multiply(a, t)
```

Comment:
supposidely this could be replaced by the `rel_entr` from scipy: https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.rel_entr.html#scipy.special.rel_entr

The difference is that we use `log2` instead of the log base e in the scipy definition. I have to check.

Reply:
So I assume that we could use the natural logarithm anywhere because it would only be different by a constant multiplier and since we are only comparing the information everywhere then it should not matter

Reply:
Okay, we had talked about this. This time I found that `scipy.special.rel_entr() `was the one that had the same results as before.